src/ProgressBar/index.jsx

```diff
@@ -116,8 +120,6 @@ ProgressBarAnnotated.propTypes = {
   variant: PropTypes.oneOf(VARIANTS),
   /** Specifies an additional `className` to add to the base element. */
   className: PropTypes.string,
-  /** Hide's the label visually. */
-  visuallyHidden: PropTypes.bool,
```

Comment:
This prop doesn't do anything for the annotated progress bar, and is currently causing a prop type warning.